packages/react/src/components/CommandBar/CommandBar.base.tsx

```diff
@@ -179,7 +179,7 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
       onClick: this._onButtonClick(item),
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28418&commitId=300ca97e60299bc85a5a1cd1b1d39fcc06cb263d&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Root - RTL.hover.chromium.png|69| Changed|
|CommandBar.Root.hover.chromium.png|69| Changed|
|CommandBar.Text only.hover.chromium.png|69| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>